petastorm/tests/test_end_to_end.py

```diff
@@ -857,3 +862,32 @@ def test_make_batch_reader_with_url_list(scalar_dataset):
             row_count += len(batch.id)
 
         assert row_count == 100
+
+
+def test_pyarrow_filters_make_reader(synthetic_dataset):
+    with make_reader(synthetic_dataset.url, workers_count=5, num_epochs=1,
+                     filters=[('partition_key', '=', 'p_5'), ]) as reader:
+        uv = set()
+        for data in reader:
+            uv.add(data[0])
```

Comment:
Can we use an explicit namedtuple name here instead of `data[0]` for readability?

Reply:
done.